app/graphql/mutations/namespace.rb

```diff
@@ -0,0 +1,5 @@
+class Mutations::Namespace < Types::BaseObject
+  def self.default_graphql_name
+    "#{name.split('::')[1..-1].join}Mutations"
```

Comment:
Do we want this to be singular
```suggestion
    "#{name.split('::')[1..-1].join}Mutation"
```

Reply:
Nah, it's a collection of mutations so plural makes sense!